mesonbuild/interpreter/dependencyfallbacks.py

```diff
@@ -340,6 +340,23 @@ def lookup(self, kwargs: TYPE_nkwargs, force_fallback: bool = False) -> Dependen
                         self._subproject_impl(subp_name, varname)
                     break
 
+                # Check if we could have a matching Rust crate imported from
+                # Cargo.lock, with expected api version. We know there is no
+                # wrap-file defined to match this crate.
+                if name.endswith('-rs'):
+                    s = name.split('-')
+                    if (len(s) < 3):
```

Comment:
if isn't a function, don't parenthesize `len(s) < 3`.

Reply:
Will fix this.